hmda-data-publisher/src/test/scala/hmda/publisher/query/component/PublisherComponent2021Spec.scala

```diff
@@ -0,0 +1,131 @@
+package hmda.publisher.query.component
+
+import akka.actor.ActorSystem
+import akka.stream.scaladsl.{Sink, Source}
+import akka.testkit.TestKit
+import hmda.publisher.query.lar._
+import hmda.publisher.query.panel.InstitutionEntity
+import hmda.query.ts.TransmittalSheetEntity
+import hmda.utils.EmbeddedPostgres
+import org.scalatest.concurrent.{PatienceConfiguration, ScalaFutures}
+import org.scalatest.time.{Millis, Minutes, Span}
+import org.scalatest.{BeforeAndAfterAll, BeforeAndAfterEach, FreeSpecLike, Matchers}
+
+import scala.concurrent.ExecutionContext.Implicits._
+import scala.concurrent.duration._
+import scala.concurrent.{Await, Future}
+
+class PublisherComponent2021Spec
+  extends TestKit(ActorSystem("PublisherComponent2021Spec"))
+    with FreeSpecLike
+    with Matchers
+    with ScalaFutures
+    with EmbeddedPostgres
+    with BeforeAndAfterAll
+    with BeforeAndAfterEach
+    with PublisherComponent2021
+    with PatienceConfiguration {
+
+  val institutionRepo = new InstitutionRepository2021(dbConfig)
+  val tsRepo          = createTransmittalSheetRepository2021(dbConfig, Year2021Period.Whole)
+  val larRepo         = createLarRepository2021(dbConfig,Year2021Period.Whole)
+  override implicit def patienceConfig: PatienceConfig = PatienceConfig(timeout = Span(2, Minutes), interval = Span(100, Millis))
+
+  override def beforeAll(): Unit = {
+    super.beforeAll()
+    Await.ready(
+      Future.sequence(
+        List(
+          institutionRepo.createSchema(),
+          tsRepo.createSchema()
+        )
+      ),
+      30.seconds
+    )
+  }
+
+  "InstitutionRepository2021 runthrough" in {
+    import institutionRepo._
+    val data = InstitutionEntity("EXAMPLE-LEI", activityYear = 2020, institutionType = 1, taxId = "ABC", hmdaFiler = true)
+    data.toPSV
```

Comment:
What is the function of the toPSV and all the toFooPSV invocations? I understand these methods return strings and leave data as is. Is something catching and testing the returned strings? It looks like the tests are based on the original "data" vals.

Reply:
I thought that PSV was a data format for mlar and it's invocation here in the tests is just to make sure it runs but I could be wrong, this was based on the previous tests that were in the codebase.